pulsar-broker/src/test/java/org/apache/pulsar/client/api/ClientDeduplicationTest.java

```diff
@@ -372,4 +375,51 @@ public void testKeyBasedBatchingOrder() throws Exception {
         consumer.close();
         producer.close();
     }
+
+    @Test
+    public void testUpdateSequenceIdInSyncCodeSe7gment() throws Exception {
+        final String topic = "persistent://my-property/my-ns/testUpdateSequenceIdInSyncCodeSegment";
+        int totalMessage = 10;
+        int threadSize = 30;
```

Comment:
it's better to decrease the `threadSize` and increase the `totalMessage`, the test CPU cores size is not large, so `threadSize` can't be set to large, a lot of threads will wait. better to increase the  `totalMessage`.